src/components/common/WalletListModalCreateWallet.js

```diff
@@ -2,20 +2,18 @@
 import React, { Component } from 'react'
 import { Image, Text, TouchableHighlight, View } from 'react-native'
 
-import localize from '../../locales/strings'
+import s from '../../locales/strings'
 import FormattedText from '../../modules/UI/components/FormattedText/index'
 import { CryptoExchangeWalletListRowStyle as styles } from '../../styles/components/CryptoExchangeWalletListRowStyle.js'
+import { type GuiWalletType } from '../../types/types.js'
 
 type Props = {
-  supportedWallet: Object,
-  onPress(Object): void
+  supportedWallet: GuiWalletType,
+  onPress(GuiWalletType): void
 }
-type State = {}
 
-class CryptoExchangeCreateWalletRow extends Component<Props, State> {
-  onPress = () => {
-    this.props.onPress(this.props.supportedWallet)
-  }
+class WalletListModalCreateWallet extends Component<Props> {
```

Comment:
I think this should be called `WalletListModalCreateRow`. This would be more consistent with the other names we have like `SettingsSwitchRow` / `SettingsHeaderRow` / `SettingsRadioRow`.

Reply:
I tried rebasing this myself, because it seemed easy, but I got a Flow error:

```js
if (showCreateWallet) {
  return <CryptoExchangeCreateWalletRow supportedWallet={createWalletCurrency || {}} ... />
```

Turns out, `|| {}` isn't a valid GuiWalletType, so I fixed the if statement instead:

```js
if (showCreateWallet && createWalletCurrency) {
  return <WalletListModalCreateRow supportedWallet={createWalletCurrency} ... />
```

Reply:
I went ahead and rebased your branch, since it seemed simple enough to do.